packages/tests/web-test-runner.config.js

```diff
@@ -164,8 +164,14 @@ export default {
   rootDir: '../',
   // Note this file list can be overridden by wtr command-line arguments.
   files: [
-    '../lit-html/development/**/*_test.(js|html)',
+    '../labs/observers/development/**/*_test.(js|html)',
```

Comment:
Need `motion` in here too

Reply:
Motion tests have never been tested in CI, and they currently fail: https://github.com/lit/lit/issues/2009

So I'd suggest handling that in a separate PR.

Reply:
ahh... god point.